var/spack/repos/builtin/packages/rmgdft/package.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rmgdft(CMakePackage, CudaPackage, ROCmPackage):
+    """RMGDFT is a high performance real-space density functional code
+       designed for large scale electronic structure calculations."""
+
+    homepage = "http://www.rmgdft.org/"
+    git      = "https://github.com/RMGDFT/rmgdft.git"
+    maintainers = ['elbriggs']
+    tags = ['ecp', 'ecp-apps']
+    version('master')
```

Comment:
Can you add tagged releases as well, and explicitly point to the fact that this is from a branch:
```suggestion
    version('master', branch='master')
```
?